lld/test/MachO/weak-definition-in-main-file.ll

```diff
@@ -0,0 +1,46 @@
+; REQUIRES: aarch64
```

Comment:
Would this work if x86_64 is not configured?
(Sorry, I haven't found failures in the official bb)

Reply:
Oh, you're right, sorry. This should say REQUIRES: x86_64 of course.

Reply:
Finally fixed in 94d9455157d2a267b0c6caf37977cb0a74763198. Thanks again.